health-services/project-factory/src/server/generateFlowClasses/boundary-generateClass.ts

```
@@ -0,0 +1,298 @@
+            }
+        }
+
+        // Step 4: Start traversal from roots
+        for (const root of roots) {
+            traverse(root);
+        }
+
+        return result;
+    }
+
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)